op-guide/dashboard-overview-info.md

```diff
@@ -5,37 +5,72 @@ category: operations
+PD | Store Status -- Up Stores | the number of TiKV nodes that are up |
+PD | Store Status -- Disconnect Stores | the number of TiKV nodes that encounter abnormal communication within a short time |
+PD | Store Status -- LowSpace Stores | the number of TiKV nodes with an available space of less than 80% |
+PD | Store Status -- Down Stores | the number of TiKV nodes that are down | The normal value is `0`. If the number is bigger than `0`, it means some node(s) are abnormal.
+PD | Store Status -- Offline Stores | the number of TiKV nodes (still providing service) that are being made offline |
+PD | Store Status -- Tombstone Stores | the number of TiKV nodes that are successfully offline |
+PD | 99% completed_cmds_duration_seconds | the 99th percentile duration to complete a pd-server request | less than 5ms
+PD | handle_requests_duration_seconds | the request duration of a PD request |
+TiDB | Statement OPS | the total number of executed SQL statements, including `SELECT`, `INSERT`, `UPDATE` and so on |
+TiDB | Duration | the execution time of a SQL statement |
+TiDB | QPS By Instance | the QPS on each TiDB instance |
+TiDB | Failed Query OPM | the number of failed SQL statements, including syntax error and key conflicts and so on |
+TiDB | Connection Count | the connection number of each TiDB instance |
+TiDB | Heap Memory Usage | the size of heap memory used by each TiDB instance |
+TiDB | Transaction OPS | the number of executed transactions per second |
+TiDB | Transaction Duration | the execution time of a transaction |
+TiDB | KV Cmd OPS | the number of executed KV commands |
+TiDB | KV Cmd Duration 99 | the execution time of the KV command |
+TiDB | PD TSO OPS | the number of TSO that TiDB obtains from PD |
+TiDB | PD TSO Wait Duration | the time consumed when TiDB obtains TSO from PD |
+TiDB | TiClient Region Error OPS | the number of Region related errors returned by TiKV |
+TiDB | Lock Resolve OPS | the number of transaction related conflicts |
+TiDB | Load Schema Duration | the time consumed when TiDB obtains Schema from TiKV |
+TiDB | KV Backoff OPS | the number of errors returned by TiKV (such as transaction conflicts )
+TiKV | leader | the number of leaders on each TiKV node |
+TiKV | region | the number of Regions on each TiKV node |
+TiKV | CPU | the CPU usage ratio on each TiKV node |
+TiKV | Memory | the memory usage on each TiKV node |
+TiKV | store size | the data amount on each TiKV node |
+TiKV | cf size | the data amount on different CFs in the cluster |
+TiKV | channel full | `No data points` is displayed in normal conditions. If a monitoring value displays, it means the corresponding TiKV node fails to handle the messages |
+TiKV | server report failures | `No data points` is displayed in normal conditions. If `Unreachable` is displayed, it means TiKV encounters a communication issue. |
```

Comment:
`No data points` -> `No data`?

Reply:
`No data points` is the exact phrase that is displayed on the dashboard, so I think we should keep it.